snippets/Google.Logging.Log4Net.Snippets/GoogleStackdriverAppenderSnippets.cs

```diff
@@ -0,0 +1,64 @@
+﻿// Copyright 2016 Google Inc. All Rights Reserved.
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+using log4net;
+using log4net.Config;
+using System.IO;
+using System.Text;
+using Xunit;
+
+namespace Google.Logging.Log4Net.Snippets
+{
+    public class GoogleStackdriverAppenderSnippets
+    {
+        private class Program { }
+
+        [Fact]
+        public void Overview()
+        {
+            // Sample: Overview
+            var xml = @"
+<?xml version=""1.0"" encoding=""utf-8"" ?>
+<log4net>
+  <appender name=""CloudLogger"" type=""Google.Logging.Log4Net.GoogleStackdriverAppender,Google.Logging.Log4Net"">
+    <layout type=""log4net.Layout.PatternLayout"">
+      <conversionPattern value=""%-4timestamp [%thread] %-5level %logger %ndc - %message""/>
+    </layout>
+    <projectId value = ""your-project-ID""/>
```

Comment:
I realize I didn't explain how I'd do that - I'd use a verbatim interpolated string, so these lines would be:

```text
<projectId value=""{projectId}"" />
<logId value=""{logId}"" />
```

where `projectId` and `logId` are variables declared earlier.